source/globalCommands.py

```diff
@@ -1139,18 +1139,7 @@ def script_review_currentCharacter(self,gesture):
 			try:
 				c = ord(info.text)
 			except TypeError:
-				# This might be a character taking multiple code points.

```

Comment:
In python 3, a character in a string is always one code point. Therefore this code is now obsolete.